api_views/books.py

```diff
@@ -9,11 +10,8 @@
 from app import vuln
 
 
-def error_message_helper(msg):
```

Comment:
Removing the error_message_helper function without providing an alternative error handling mechanism could lead to inconsistent error responses across the API.